pkg/queue/health/health_state.go

```diff
@@ -81,9 +81,11 @@ func (h *State) drainFinished() {
 
 }
 
-// HealthHandler constructs a handler that returns the current state of
-// the health server.
-func (h *State) HealthHandler(prober func() bool) func(w http.ResponseWriter, r *http.Request) {
+// HealthHandler constructs a handler that returns the current state of the
+// health server. If isAggressive is true and prober has succeeded previously,
+// will immediately return success without probing user-container again (until
```

Comment:
```suggestion
// the function will immediately return success without probing user-container again (until
```

Reply:
It feels that `isAggressive` is the opposite of its meaning. Since if it's `true`, then we don't probe again, if it's false we probe aggressively.

Reply:
good point. Will update it

Reply:
`Aggressive` was intended to refer to the way it hammers retries. 🤷‍♂️